tests/unit-tests/test_config.py

```diff
@@ -480,6 +480,7 @@ def test_train_info():
         check_failure(config, "batch_size")
         config._batch_size = 32
         assert config.batch_size == 32
+        config._eval_batch_size = 32
         assert config.eval_batch_size == 32
```

Comment:
```suggestion
        assert config.eval_batch_size == 10000
```
You need to test eval_batch_size default return value as 10000.